code/API_definitions/device-roaming-status-subscriptions.yaml

```diff
@@ -38,13 +37,21 @@ info:
 
     Note: Additionally to these list, ``org.camaraproject.roaming-status-subscriptions.v0.subscription-ends`` notification `type` is sent when the subscription ends.
     This notification does not require dedicated subscription.
-    It is used when the subscription expire time (optionally set by the requester) has been reached or if the API server has to stop sending notification prematurely.
+
+    It is used when:
```

Comment:
Rather "It is used in following cases" to emphasize that those lines are OR connected